src/assets/scss/_gift-config.scss

```diff
@@ -65,11 +65,13 @@ label.btn-radio {
   }
 
   &:hover input[type="radio"] ~ i.fa {
+    /* stylelint-disable-next-line */
```

Comment:
Are these here because there is no variable?

Reply:
Yes. The rule sees if there are any HEX values and creates a warning. So if there isn't a variable, we have to use this ignore to prevent the warning.

I would prefer the rule to check the variables and then the styles, so we so we don't have to add this ignore. But it is still good as it pushes devs to use variables as much as they can.

Reply:
I concur. Writing a new HEX should be the exception, not the rule, so there shouldn't be many of these ignores in a good, color-consistent design.